src/native/corehost/test/nativehost/host_context_test.cpp

```diff
@@ -1000,3 +1028,13 @@ bool host_context_test::app_get_function_pointer(
 
     return app_get_function_pointer_test(hostfxr, argc, argv, config_log_prefix, test_output);
 }
+
+bool host_context_test::get_runtime_delegate(
```

Comment:
I would prefer to use more specific naming - `get_runtime_delegate_for_active_context` or something like that. And similarly rename the implementation method and the string passed to the helper to run it.